sale_order_product_recommendation/wizards/sale_order_recommendation_view.xml

```diff
@@ -26,6 +26,7 @@
                                 <field name="times_delivered" />
                                 <field name="units_delivered" />
                                 <field name="units_included" widget="numeric_step" />
+                                 <field name="sale_uom_id" optional="show" />
```

Comment:
Wasn't this handled on the cherry-picked commit?

Reply:
Not yet. the PR continues in open state: https://github.com/OCA/sale-workflow/pull/2065